Source/WebKit/GPUProcess/media/RemoteAudioSessionProxyManager.cpp

```diff
@@ -188,6 +224,9 @@ bool RemoteAudioSessionProxyManager::tryToSetActiveForProcess(RemoteAudioSession
     if (!hasActiveNotInterruptedProxy()) {
         // This proxy and only this proxy wants to become active. Activate
         // the session, and return whether that activation was successful.
+#if PLATFORM(IOS_FAMILY)
+        providePresentingApplicationPID(proxy);
```

Comment:
I wonder what happens in the following use case:
- A first presenting app/web page using GPUProcess G starts camera/microphone capture (or maybe audio context in media playback session type).
- A second presenting app/web page using G starts an AudioContext. It wants to activate the audio session in ambient.
- The first presenting app/web page may or may not be interrupted.

The presenting application PID will likely stay to the one of the first app, while I think it would move to the second app before the PR. I am not sure whether we want to handle this case.